Dockerfile_CUDA11

```diff
@@ -0,0 +1,28 @@
+# FROM nvidia/cuda:10.1-cudnn7-devel
+FROM nvidia/cuda:11.1.1-cudnn8-devel-ubuntu18.04
+
+# https://github.com/NVIDIA/nvidia-docker/issues/1632
+#RUN rm /etc/apt/sources.list.d/cuda.list
+#RUN rm /etc/apt/sources.list.d/nvidia-ml.list
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update && apt-get install -y \
+	python3-opencv ca-certificates python3-dev git wget sudo ninja-build
+RUN ln -sv /usr/bin/python3 /usr/bin/python
+
+# create a non-root user
+COPY . /home/appuser
+WORKDIR /home/appuser
+
+# https://github.com/facebookresearch/detectron2/issues/3933
+ENV PATH="/home/appuser/.local/bin:${PATH}"
+RUN wget https://bootstrap.pypa.io/pip/3.6/get-pip.py && \
+	python3 get-pip.py && \
+	rm get-pip.py
+
+# install dependencies
+# See https://pytorch.org/ for other options if you use a different version of CUDA
+RUN pip install tensorboard cmake   # cmake from apt-get is too old
+RUN pip install torch==1.10 torchvision==0.11.1 -f https://download.pytorch.org/whl/cu111/torch_stable.html
+# RUN pip install torch==1.6.0+cu101 torchvision==0.7.0+cu101 -f https://download.pytorch.org/whl/cu101/torch_stable.html
+# RUN pip install -i https://pypi.tuna.tsinghua.edu.cn/simple tensorboard opencv-python cython yacs termcolor scikit-learn tabulate gdown gpustat faiss-gpu ipdb h5py
```

Comment:
I removed `https://pypi.tuna.tsinghua.edu.cn/simple`. Do we need this ? @L1aoXingyu